.github/workflows/build.yml

```diff
@@ -20,6 +20,12 @@ jobs:
         - '6.1.5'
         - '7.0.2.3'
         - '7.1.0'
+        - '7.2.0.rc1'
+        exclude:
+        - ruby-version: 2.7
```

Comment:
Rails 7.2 only supports Ruby 3.1+.
https://edgeguides.rubyonrails.org/7_2_release_notes.html#make-ruby-3-1-the-new-minimum-version